log/logger.py

```diff
@@ -194,12 +187,17 @@ def setup(self, log_level: int = logging.DEBUG, file_log_level: int = logging.DE
             file_handler.setLevel(file_log_level)
 
             # create StreamHandler
-            # default is without param, than the handler is sys.stderr
-            stream_handler = logging.StreamHandler(sys.stdout)
+            stream_handler = logging.StreamHandler()
             stream_handler.setLevel(stream_log_level)
 
+            # create ConsoleHandler
+            # '/dev/tty'
+            _console = open('/dev/console', 'w')
```

Comment:
vorschlag: 
- Verbindung zur tty console aus dem Setup auslagern, besser eigene methode. User soll aktivieren/deaktivieren können
- Sieht statisch aus, somit kann ich mir die Log nicht in einem beliebigen Terminal anzeigen oder?